etc/encoding/opencast.properties

```diff
@@ -0,0 +1,243 @@
+  -filter:v scale=w=854:h='trunc(854/dar/2)*2',setsar=1,fps=25 -pix_fmt yuv420p \
+    -c:v libx264 -profile:v high -level 4.0  -crf 23 \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -maxrate 800k -bufsize 1600k \
+    -c:a aac -b:a 64k -ac 2 \
+    -movflags faststart #{out.dir}/#{out.name}#{out.suffix.480p-quality} \
+  -filter:v scale=w=1280:h='trunc(1280/dar/2)*2',setsar=1,fps=25 -pix_fmt yuv420p \
+    -c:v libx264 -profile:v high -level 4.0  -crf 23 \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -maxrate 1200k -bufsize 2400k \
+    -c:a aac -b:a 96k -ac 2 \
+    -movflags faststart #{out.dir}/#{out.name}#{out.suffix.720p-quality} \
+  #{if-height-geq-900} \
+  #{if-height-geq-1260} \
+  #{if-height-geq-1800}
+
+# Preview image for the player, shown before the movie is started
+profile.player-preview.http.name = cover image for engage
+profile.player-preview.http.input = visual
+profile.player-preview.http.output = image
+profile.player-preview.http.suffix = -player.jpg
+profile.player-preview.http.ffmpeg.command = -i #{in.video.path} -frames:v 1 \
+  -filter:v select=eq(n\\,1)+lt(t\\,#{time}),fps=1,reverse,scale=-2:720 \
+  #{out.dir}/#{out.name}#{out.suffix}
+
+# Slide preview images as shown in the player
+profile.player-slides.http.name = slide preview image for engage
+profile.player-slides.http.input = visual
+profile.player-slides.http.output = image
+profile.player-slides.http.suffix = .#{time}.jpg
+profile.player-slides.http.ffmpeg.command = -ss #{time} -i #{in.video.path} \
+  -r 1 -frames:v 1 -filter:v yadif,scale=160:-1 \
+  #{out.dir}/#{out.name}#{out.suffix}
+
+# Cover image for search results
+profile.search-cover.http.name = cover image for engage
+profile.search-cover.http.input = visual
+profile.search-cover.http.output = image
+profile.search-cover.http.suffix = -search.jpg
+profile.search-cover.http.ffmpeg.command = -i #{in.video.path} -frames:v 1 \
+  -filter:v select=eq(n\\,1)+lt(t\\,#{time}),fps=1,reverse,scale=160:-2 \
+  #{out.dir}/#{out.name}#{out.suffix}
+
+profile.mp4-preview.http.name = preview video
+profile.mp4-preview.http.input = visual
+profile.mp4-preview.http.output = visual
+profile.mp4-preview.http.suffix = -preview.mp4
+profile.mp4-preview.http.ffmpeg.command = -i #{in.video.path} \
+  -filter:v yadif,scale=-2:360 \
+  -c:v libx264 -preset veryfast -crf 23 -profile:v baseline -pix_fmt yuv420p -tune film \
+  -c:a aac -strict -2 -ar 22050 -ab 64k \
+  -movflags faststart #{out.dir}/#{out.name}#{out.suffix}
+
+profile.mp4-preview.dual.http.name = preview video (picture-by-picture)
+profile.mp4-preview.dual.http.input = visual
+profile.mp4-preview.dual.http.output = visual
+profile.mp4-preview.dual.http.suffix = -preview-composite.mp4
+profile.mp4-preview.dual.http.ffmpeg.command = -i #{in.video.path} \
+  #{compositeCommand} \
+  -c:v libx264 -preset veryfast -crf 23 -profile:v baseline -pix_fmt yuv420p -tune film \
+  -c:a aac -strict -2 -ar 22050 -ab 64k \
```

Comment:
Why `-strict -2`? That shouldn't be necessary any longer (was years ago when the `aac` encoder was marked experimental).